### ISSUE TITLE
Implement Age 2 cards for Artifacts Expansion

### ISSUE THREAD

**Issue by micahstairs** [micahstairs/bga-innovation#88]
This issue will track the implementation of all Age 2 cards for the Artifacts expansion. Please update this comment before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

  - [x] **125 - Seikilos Epitaph**
  - [x] **126 - Rosetta Stone**
  - [x] **127 - Chronicle of Zuo**
  - [x] **128 - Babylonian Chronicles**
  - [x] **129 - Holy Lance**
  - [x] **120 - Baghdad Battery**
  - [x] **131 - Holy Grail**
  - [x] **132 - Terracotta Army**
  - [x] **133 - Dead Sea Scrolls**
  - [x] **134 - Cyrus Cylinder**
